src/editor-service.ts

```diff
@@ -13,8 +13,8 @@
 
 import {SourcePosition, SourceRange, Warning} from 'polymer-analyzer';
 
-export type TypeaheadCompletion =
-    ElementCompletion | AttributesCompletion | AttributeValuesCompletion;
+export type TypeaheadCompletion = ElementCompletion | AttributesCompletion |
+    AttributeValuesCompletion | DatabindingPropertiesCompletion;
 export interface ElementCompletion {
   kind: 'element-tags';
   elements: {
```

Comment:
Hm, yeah, that formatting is kinda unfortunate. I couldn't find a good format without breaking the individual element completions out into their own type, so I did that.